system-helper/flatpak-system-helper.c

```diff
@@ -237,16 +322,46 @@ handle_deploy (FlatpakSystemHelper   *object,
   gboolean local_pull;
   gboolean reinstall;
   g_autofree char *url = NULL;
+  g_autoptr(OngoingPull) ongoing_pull = NULL;
+  g_autofree gchar *src_dir = NULL;
+  gint dir_fd = -1;
 
   g_debug ("Deploy %s %u %s %s %s", arg_repo_path, arg_flags, arg_ref, arg_origin, arg_installation);
-
-  system = dir_get_system (arg_installation, get_sender_pid (invocation), &error);
+  system = dir_get_system ("", get_sender_pid (invocation), &error);
```

Comment:
I think this change from `arg_installation` to `""` is a leftover development change?